.github/workflows/docker_test.yml

```diff
@@ -1,19 +1,55 @@
 name: Test Docker images
 
 on:
+  workflow_call:
+    inputs:
+      image_name:
+        required: true
+        type: string
+      tag_name:
+        required: true
+        type: string
+      nightly_tag_main:
```

Comment:
Can we make this docker_test generic to `image_name` and `tag`? Why do we need a separate `nightly`?

Reply:
see my other comment for clarification / correction.